1-exercises/B-array-some/exercise.js

```diff
@@ -12,6 +12,11 @@ let pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 // https://nodejs.org/api/process.html#process_process_exit_code
 // process.exit(1);
 
+function isNull(arr){
+  return arr.some(()=>{return arr.includes(null)})
+}
+if(isNull(pairsByIndex)) return;
```

Comment:
good, but remember to use process.exit(1) as in instructions if a null value is found :)